saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py

```diff
@@ -6,42 +6,68 @@
 
 from .....channel.models import Channel
 from .....giftcard.models import GiftCard
+from .....graphql.tests.queries import subscription_queries
```

Comment:
`subscription_queries.py` could be moved to `tests/subscription_webhooks` I guess? I think it won't be needed anywhere else in Saleor.

Reply:
moved